puppet-client.html

```diff
@@ -239,7 +232,8 @@
             }
         },
         onLocalChange: function(patches) {
-            this.propagateDomBind(patches);
+            // todo #26
+            this.propagateDomBind({patches: patches, results: patches.map(function() { return {};})});
```

Comment:
Plus if that is ppc talking to ppc there is no API been violated, as there is no P in this interface. So poor ppc should smile and digest it's own objects further down.